randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -1583,25 +1625,16 @@ Extra - asset_id: collision_camera_021
 > Ledge Below PB Tank; Heals? False
   * Layers: default
   > Door to Above Pulse Radar
-      Simple IBJ or Use Spin Boost
+      Movement (Beginner) or Simple IBJ or Use Spin Boost
```

Comment:
Not sure what the "Movement (Beginner)" method is here - is it a slide jump? Could you add a comment clarifying this?

Reply:
you can jump to the edge, but it's not a jump that is very simple. the jump doesnt even look doable, but it is

Reply:
i also tested this a while ago back when i first did the revision. i could try again and see if it should just be a trivial connection